### PR TITLE
fix: allowWatchBookmarks should be configurable

### PR DESCRIPTION
**Pull request by cmwylie19**
Make allowWatchBookmarks configurable

### CODE REVIEW

File: src/fluent/watch.ts [hunk 1]
```diff
@@ -352,7 +364,10 @@ export class Watcher<T extends GenericClass> {
       // Otherwise, call the finally function if it exists
       this.#events.emit(WatchEvent.GIVE_UP, err);
       this.close();
+      // await this.#runner();
```
Comment (cmwylie19) [inline src/fluent/watch.ts:367]:
```suggestion
```

File: src/fluent/watch.ts [hunk 2]
```diff
@@ -352,7 +364,10 @@ export class Watcher<T extends GenericClass> {
...
     }
+    // // Retry the watch after the delay
+    // await this.#runner();
```
Comment (cmwylie19) [inline src/fluent/watch.ts:370]:
```suggestion
```

File: src/fluent/watch.ts [hunk 3]
```diff
@@ -39,6 +39,9 @@ export enum WatchEvent {
 
 /** Configuration for the watch function. */
 export type WatchCfg = {
+  // allowWatchBookMarks -> allowWatchBookmarks
+  /** Whether to allow watch bookmarks. */
+  allowWatchBookMarks?: boolean;
```
Comment (cmwylie19) [inline src/fluent/watch.ts:44]:
```suggestion
  /** Whether to allow watch bookmarks. */
  allowWatchBookmarks?: boolean;
```

File: src/fluent/watch.ts [hunk 4]
```diff
@@ -191,7 +201,7 @@ export class Watcher<T extends GenericClass> {
     }
 
     // Enable watch bookmarks
-    url.searchParams.set("allowWatchBookmarks", "true");
+    url.searchParams.set("allowWatchBookmarks", this.#watchCfg.allowWatchBookMarks ? `${this.#watchCfg.allowWatchBookMarks}` : "true" );
```
Comment (cmwylie19) [inline src/fluent/watch.ts:204]:
```suggestion
    url.searchParams.set("allowWatchBookmarks", this.#watchCfg.allowWatchBookmarks ? `${this.#watchCfg.allowWatchBookmarks}` : "true" );
```

**Review by btlghrants — DISMISSED** [review]
:+1:

**Review by btlghrants — APPROVED** [review]
Yep.  Now with 50% more formatting!

**Pull request merged**